

RRTStarPlanner:
  # rrt star parameters
  goal_tolerance: 0.2
  rewiring_radius: 0.2
  epsilon: 0.3
  max_num_nodes: 5000
  # WOA Parameters
  sampling_radius: 0.15
  num_iterations: 700
  num_agents: 80
  spiral_shape: 0.9